src/utilities/createRouter.ts

```diff
@@ -20,9 +18,8 @@ type RouterUpdateOptions = {
   replace?: boolean,
 }
 
-export function createRouter<const T extends Routes>(routes: T, options: RouterOptions = {}): Router<T> {
-  const routerRoutes = createRouterRoutes(routes)
-  const resolve = createRouterResolve(routerRoutes)
+export function createRouter<const T extends RouterRoutes>(routes: T, options: RouterOptions = {}): Router<T> {
+  const resolve = createRouterResolve(routes)
```

Comment:
I love that now we accept `routes` and then we can just use them directly.

Reply:
💯